cookbook/case_studies/ml_training/house_price_prediction/multiregion_house_price_predictor.py

```diff
@@ -1,19 +1,35 @@
 """
-Predicting House Price in Multiple Regions with XGBoost and Dynamic Workflows
-------------------------------------------------------------------------------
+Predicting House Price in Multiple Regions Using XGBoost and Dynamic Workflows
+-------------------------------------------------------------------------------
+`XGBoost <https://xgboost.readthedocs.io/en/latest/>`__ is an optimized distributed gradient boosting library designed to be efficient, flexible, and portable. 
+It uses `gradient boosting <https://en.wikipedia.org/wiki/Gradient_boosting>`__ technique to implement Machine Learning algorithms.
 
-In this example, you'll use the house price prediction model for one region to expand it to multiple regions. 
+In this tutorial, we will understand how to predict house prices in multiple regions using XGBoost, and dynamic workflows in Flyte.
+
+A dynamic workflow is a typical workflow where the users can perform any arbitrary computation by consuming the inputs and producing the outputs. In the backend, it is designed as a task. During execution, the function body can be run to produce a workflow.
```

Comment:
I think we don't really need this cause we already talked about dynamic workflows in the README.

Reply:
sure, we have only mentioned about iterating through the regions and building a DAG in the README. If you think that is enough, I will remove this. Should I add it to README otherwise?

Reply:
Yeah, I think that's enough. You can link the dynamic workflows text to the doc page we have on it.